packages/pf-codemods/README.md

```diff
@@ -480,9 +480,9 @@ Out:
 <SimpleList isActive />
 ```
 
-### spinner-svg-default [(#8183)](https://github.com/patternfly/patternfly-react/pull/8183)
+### spinner-svg-default [(#8616)](https://github.com/patternfly/patternfly-react/pull/8616)
```

Comment:
Nit: since we're updating what the codemod does, we could update the file names/heading here to refer to the fact that the SVG prop is being removed.